nevergrad/functions/functionlib.py

```diff
@@ -238,7 +238,10 @@ def function_from_transform(self, x: np.ndarray) -> float:
         results = []
         for block in x:
             results.append(self._func(block))
-        return float(self._aggregator(results))
+        try:
+            return float(self._aggregator(results))
+        except OverflowError:
+            return 1.0e20
```

Comment:
how about float("inf") instead, this way optimizser deal with it the way they want.

Reply:
A bit zfraid that some optimization methods will crash. Max-float might also be risky.

Reply:
https://github.com/facebookresearch/nevergrad/blob/93fd71736be88eec272188436919aa0d06cfbf11/nevergrad/optimization/base.py#L323-L331

It should not

Reply:
You're right. Modified accordingly.